<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="sanket" failures="1" tests="2" name="Test" time="7.819" errors="0" timestamp="2023-12-21T19:30:55 IST">
  <testcase classname="test.Testt" name="checkErrorMassage" time="1.882"/>
  <testcase classname="test.Testt" name="makeErrorMassageFailWithWrongExpResult" time="2.091">
    <failure type="java.lang.AssertionError" message="expected [Sorr, your password was incorrect. Please double-check your password.] but found [Sorry, your password was incorrect. Please double-check your password.]">
      <![CDATA[java.lang.AssertionError: expected [Sorr, your password was incorrect. Please double-check your password.] but found [Sorry, your password was incorrect. Please double-check your password.]
at test.Testt.makeErrorMassageFailWithWrongExpResult(Testt.java:98)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- makeErrorMassageFailWithWrongExpResult -->
</testsuite> <!-- Test -->
